compatibility-guidelines.md

```diff
@@ -0,0 +1,176 @@
+# Introduction
+
+An independent third party researched how other leading blockchain networks
+manage breaking changes and backwards compatibility.
+
+The independent 3rd party concluded the following:
+
+* None of the leading blockchain networks have a definitive public policy with
+  respect to breaking changes and backwards compatibility.   
+* None have stated that breaking changes cannot happen. All networks
+  communicate an intent to prioritize backwards compatibility and propose that
+  major changes and deprecations will be communicated clearly and in advance.
+
+In summary, all examined networks:
+
+* Strive to balance technological advancement with maintaining stability and
+  minimizing disruption to existing applications and services.  
+* Employ various communication channels, including release notes, blog posts,
+  and developer forums, to inform stakeholders of changes.  
+* Utilize governance mechanisms involving validators and/or token holders to
+  approve major protocol changes.
+* Provide migration guides and support during significant updates or SDK
+  deprecations to facilitate smooth transitions for developers.
+
+Hiero intends to follow the same principles.
+
+# Backwards compatibility and breaking changes
+
+The development of software involves ongoing agility and the need to adapt to
+changing technologies and evolving user demands. The adoption of new features
+and improved capabilities often leads to critical decisions including how to
+maximize the opportunities offered by these changes against potential user
+impact. 
+
+As much as is possible, Hiero projects are committed to maintaining backwards
+compatibility to ensure a reliable experience for users and developers. Below
+is an outline of the approach towards backwards compatibility, the process for
+introducing breaking changes when necessary, and the communication and
+deprecation timelines required to maintain transparency and stability within
+the Hiero ecosystem.
+
+Hiero encompasses a large number of projects at different stages of
+development, the principles outlined below may not immediately apply to every
+Hiero project, but will apply to the following projects. Other Hiero projects
+may choose how to handle backwards compatibility as their maturity increases.
+
+* Hiero Consensus Node  
+* Hiero Block Node  
+* Hiero JSON RPC Relay  
+* Hiero Mirror Node  
+* Hiero SDKs (Go, JavaScript, Python, Rust, Java, CPP, Swift)
```

Comment:
Should we include Python SDK already today?

Reply:
Python SDK doesn't have to be included. It's not in a stable version currently. It's ok to make breaking changes until its at least at 1.x.x version.

Reply:
```suggestion
* Hiero SDKs (Go, JavaScript, Rust, Java, CPP, Swift)
```